src/lib/snack-bar/snack-bar-container.html

```diff
@@ -0,0 +1 @@
+<template portalHost></template>
```

Comment:
Add a wrapper div with the following:
```html
<div aria-live="assertive" aria-atomic="true" role="alert">
```

Eventually this needs to use `MdLiveAnnouncer`, but this can suffice for now.